handling-incidents.md

```diff
@@ -0,0 +1,101 @@
+## Handling Standards of Conduct Incidents
+
+This document is for event organizers and volunteers and provides guidelines
+on handling violations of the standards of conduct as well as incident
+complaints from attendees.
+
+## SoC Event Committee
+
+All TPF events will have a per-event Standards of Conduct committee appointed
+to act as the final authority for all issues related to the standards of
+conduct for each event. The membership of this committee may vary from event
+to event.
+
+This committee will typically include a mix of Perl Foundation board members,
+conference organizers, and community volunteers.
+
+The committee for {$event} is:
+
+* {$member_one}
+* {$member_two}
+* ...
+
+## Warnings
+
+Any event organizer or volunteer can issue a verbal warning to a participant,
+letting them know that their behavior violates the event’s standards of
+conduct. Warnings should be reported to event organizers as soon as
+practical. The report should include:
+
+* Identifying information about the participant(s) (name, appearance,
+  clothing, etc.) given a warning.
+* The time of day the warning was issued.
+* The behavior that was in violation.
+* The approximate time of the behavior (if different than the time of
+  warning).
+* The circumstances surrounding the incident.
+* Your identity.
+* Other people involved in the incident.
+
+Please send all reports by email to
+[mailto:soc@perlconference.us](soc@perlconference.us).
+
+## Presentations
+
+Presentations or similar events should not be stopped for one-time gaffes or
+minor problems, although a member of the event organizer or volunteers should
+speak to the presenter afterward. However, organizers and volunteers should
+take immediate action to politely and calmly stop any presentation or event
+that repeatedly or seriously violates the anti-harassment policy. For example,
+simply say "I’m sorry, this presentation cannot be continued at the present
+time" and provide no further explanation.
+
+## Taking reports
+
+Whenever possible, event organizers or volunteers should direct complaints to
+a member of the SoC Event Committee. However, that may not always be possible,
+and organizers or volunteers should be prepared to handle an incident report.
+
+When taking a report from someone experiencing harassment, you should record
+what they say and reassure them they are being taken seriously, but avoid
+making specific promises about what actions will be taken in response. Ask for
+any other information if the reporter has not volunteered it (such as time and
+place) but do not pressure them to provide it if they are reluctant. If the
+reporter desires it, arrange for an escort by an organizer, volunteer, or a
+trusted person, contact a friend, and contact local law enforcement. Do not
+pressure the reporter to take any action if they do not want to do it.
+
```

Comment:
Perhaps a mention in here somewhere that we will typically not act on an Anonymous complaint alone.    We will need to have at least once witness to the incident that is willing to go on the record or have some sort of evidence of the incident before we can approach the accused.

Reply:
Done.